### ISSUE TITLE
Feature request: instal Sublime project dependencies with one command

### ISSUE THREAD

**Issue by serhiipalash** [wbond/package_control#1406]
Hi! Thank you for this great plugin!

I want to automate packages installation for our project. The idea is that we hire a new guy and all he needs to do is to run one command in Sublime Package Control and then he gets the standard development environment for our team. The command name could be `Install Project Dependencies`. Required packages could be defined in `<project_name>.sublime-project` file under `packages` key.

What do you think, is this a good idea? Maybe someone already asked for this and you can contact me with this person, so we can create a pull request?

**Comment by deathaxe** [comment]
I don't think it's feasable to install/uninstall packages on a per-project basis as several projects may be open at the same time in several windows.

Deploying a _Packages/User/Package Control.sublime-settings_ with a list of `"installed_packages": [...]` should be enough to get new users started with a pre-defined set of packages.

Other than that `Package Control: Advanced Install Package` can be used in PC 3.x.

All relevant commands to handle installation, upgrade, removal, enabling or disabling packages will be available in PC 4.0 and can be executed using `subl`.

#### Install Packages

```sh
subl -b --command 'install_packages {"packages": ["package1", "package2"]}'
```

#### Upgrade Packages

```sh
subl -b --command 'upgrade_packages {"packages": ["package1", "package2"]}'
```

#### Upgrade All Packages

```sh
subl -b --command 'upgrade_all_packages'
```

#### Remove Packages

```sh
subl -b --command 'remove_packages {"packages": ["package1", "package2"]}'
```

#### Enable Packages

```sh
subl -b --command 'enable_packages {"packages": ["package1", "package2"]}'
```

#### Disable Packages

```sh
subl -b --command 'disable_packages {"packages": ["package1", "package2"]}'
```

**Comment by deathaxe** [comment]
Related with https://github.com/wbond/package_control/issues/1633
